src/main/java/dev/ai4j/openai4j/StreamingRequestExecutor.java

```diff
@@ -167,7 +167,9 @@ public void onEvent(EventSource eventSource, String id, String type, String data
                 }
 
                 if ("[DONE]".equals(data)) {
-                    streamingCompletionCallback.run();
+                    if (logStreamingResponses) {
+                        log.debug("[DONE] event");
```

Comment:
this is unnecessary as it is logged already on line 166

Reply:
Makes sense. Removed.